pkg/virt-launcher/virtwrap/api/schema_test.go

```diff
@@ -21,16 +21,27 @@ package api
 
 import (
 	"encoding/xml"
+	"runtime"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 )
 
-var exampleXML = `<domain type="kvm" xmlns:qemu="http://libvirt.org/schemas/domain/qemu/1.0">
+func getExampleXMLForArch() string {
+	var exampleXMLArch = `<domain type="kvm" xmlns:qemu="http://libvirt.org/schemas/domain/qemu/1.0">
   <name>mynamespace_testvmi</name>
   <memory unit="MB">9</memory>
-  <os>
-    <type arch="x86_64" machine="q35">hvm</type>
+  <os>`
```

Comment:
The same here. Let's copy in the whole domain xml ...

Reply:
Ok